samples/bluetooth/central_dfu_smp/README.rst

```diff
@@ -0,0 +1,98 @@
+.. _bluetooth_central_dfu_smp:
+
+Bluetooth: Central DFU SMP
+##########################
+
+The Central DFU SMP demonstrates how to use the :ref:`dfu_smp_c_readme`.
+It uses DFU SMP Client to connect and send simple echo command.
+Ther response is then received and the received CBOR encoded data decoded and printed.
+
+
+Overview
+********
+
+After the connection the sample starts MTU size negotiaton.
+The device is enumerated and the DFU SMP Client is configured.
```

Comment:
What does _"The device is enumerated"_ exactly mean? The GATT database of the connected device is discovered?

Reply:
Same question here - please check if my update makes any sense.

Reply:
Look good to me.

Reply:
Ok